--- dsdt.dsl    2025-11-26 12:41:31.399421614 -0800
+++ dsdt.dsl    2025-11-26 12:42:49.468435148 -0800
@@ -18,12 +18,12 @@
  *     Compiler ID      "INTL"
  *     Compiler Version 0x20230331 (539165489)
  */
-DefinitionBlock ("", "DSDT", 2, "_ASUS_", "Notebook", 0x01072009)
+DefinitionBlock ("", "DSDT", 2, "_ASUS_", "Notebook", 0x0107200A)
 {
     External (_SB_.ALIB, MethodObj)    // 2 Arguments
     External (_SB_.APTS, MethodObj)    // 1 Arguments
     External (_SB_.ATKD.MAKY, UnknownObj)
-    External (_SB_.ATKD.WMNB.M009, MethodObj)    // 1 Arguments
+    External (M009, MethodObj)    // 1 Arguments
     External (_SB_.AWAK, MethodObj)    // 1 Arguments
     External (_SB_.GSRV, MethodObj)    // 1 Arguments
     External (_SB_.LID0, DeviceObj)
@@ -14497,8 +14497,8 @@

         Method (_QB6, 0, NotSerialized)  // _Qxx: EC Query, xx=0x00-0xFF
         {
-            Sleep (0x64)
-            STPL ()
+            // Sleep (0x64)
+            // STPL ()
         }

         Method (_QD4, 0, NotSerialized)  // _Qxx: EC Query, xx=0x00-0xFF
